src/chunk.c

```diff
@@ -2266,6 +2266,18 @@ ts_chunk_get_by_relid(Oid relid, bool fail_if_not_found)
 
 	schema = get_namespace_name(get_rel_namespace(relid));
 	table = get_rel_name(relid);
+
+	if (schema == NULL || table == NULL)
+	{
+		if (fail_if_not_found)
+			ereport(ERROR,
+					(errcode(ERRCODE_UNDEFINED_OBJECT),
+					 errmsg("chunk not found"),
+					 errdetail("relation with OID %u does not exist", relid)));
```

Comment:
Nit: only main error should start with lower case letter. Details and hints start with upper case (this is according to PostgreSQL error style guide).

```suggestion
					 errdetail("Relation with OID %u does not exist", relid)));
```

Reply:
Not needed now.